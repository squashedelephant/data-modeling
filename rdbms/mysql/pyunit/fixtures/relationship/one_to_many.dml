INSERT INTO users(name) VALUES ('Bill Gates');
SET @user_id = LAST_INSERT_ID();
INSERT INTO phone_numbers(user_id, phone_number) VALUES (@user_id, '425-555-1212');
INSERT INTO phone_numbers(user_id, phone_number) VALUES (@user_id, '425-111-2222');
INSERT INTO users(name) VALUES ('Steve Jobs');
SET @user_id = LAST_INSERT_ID();
INSERT INTO phone_numbers(user_id, phone_number) VALUES (@user_id, '408-555-1212');
INSERT INTO phone_numbers(user_id, phone_number) VALUES (@user_id, '408-333-4444');
INSERT INTO users(name) VALUES ('Larry Ellison');
SET @user_id = LAST_INSERT_ID();
INSERT INTO phone_numbers(user_id, phone_number) VALUES (@user_id, '650-555-1212');
INSERT INTO phone_numbers(user_id, phone_number) VALUES (@user_id, '650-555-6666');
