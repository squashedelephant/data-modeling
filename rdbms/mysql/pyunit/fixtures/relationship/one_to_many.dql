SELECT u.name, p.phone_number FROM users AS u INNER JOIN phone_numbers AS p ON u.id = p.user_id;
