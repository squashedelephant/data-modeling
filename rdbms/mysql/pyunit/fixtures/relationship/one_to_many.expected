Bill Gates
425-555-1212
Bill Gates
425-111-2222
Steve Jobs
408-555-1212
Steve Jobs
408-333-4444
Larry Ellison
650-555-1212
Larry Ellison
650-555-6666
